src/routes/search.js

```diff
@@ -240,6 +249,33 @@ class SearchRoute extends Component {
     this.search({ ...params, page });
   };
 
+  notFoundMessage = (searchType) => {
+    const { params } = this.state;
+    let idWithQuery; let idnoQuery;
```

Comment:
a really really minor nitpick - can 'idnoQuery' case be changed to idNoQuery

Reply:
Thx -- good suggestion -- I ended up instead moving lookup of translation keys to a separate function as suggested above